docs/pages/deploy-a-cluster/teleport-enterprise/gcp-kms.mdx

```diff
@@ -0,0 +1,270 @@
+---
+title: Google KMS
+description: Configure Teleport to store CA private keys in the Google Cloud Key Management Service
+h1: Store Teleport Private Keys in Google Cloud KMS
+---
+
+This guide will show you how to set up your Teleport Cluster to use the Google
+Cloud Key Management Service (KMS) to store and handle the CA private key
+material used to sign all certificates issued by your Teleport cluster.
+
+Teleport generates User, Host, JWT, and Database Certificate Authorities (CAs)
```

Comment:
We'll eventually add a desktop CA as well. Would it be best to write this sentence in such a way that it doesn't need to be updated every time?